CHANGELOG.md

```diff
@@ -57,6 +57,40 @@ Swift 5.0
 Swift 4.2
 ---------
 
+* [SE-0185][]
+
+  Protocol conformances can now be synthesized in extensions in the same file as
+  the type definition, allowing automatic synthesis of conditional conformances
+  to `Hashable`, `Equatable` and `Codable` (both `Encodable` and
+  `Decodable`). For instance,
+
+  ```swift
+  struct Generic<Param> {
+      var property: Param
+  }
+  extension Generic: Equatable where Param: Equatable {}
```

Comment:
If this last part is the auto-synthesized part, it's not clear. You could put it into a comment with something like:
// Automatically synthesized:
//   extension Generic...

Reply:
Added:

    // Automatically synthesized inside the extension:
    // static func ==(lhs: Generic, rhs: Generic) -> Bool {
    //   return lhs.property == rhs.property
    // }